src/terra/Command/Environment/EnvironmentEnable.php

```diff
@@ -70,7 +70,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         // Write drush alias.
-        $drush_alias_file_path = "{$_SERVER['HOME']}/.drush/{$app_name}.aliases.drushrc.php";
+        $drush_alias_file_path = "{$_SERVER['HOME']}/.drush/terra.{$app_name}.aliases.drushrc.php";
```

Comment:
It's fine this time.  

When you see stuff like this just commit it separately. No need for a new PR, but I would be able to cherry-pick it out to master if it was in a separate commit.